lms/templates/courseware/gradebook.html

```diff
@@ -118,23 +120,25 @@
         </tbody>
       </table>
     </div>
-    <span class="grade-book-footer">
-        %if page["previous_offset"] is not None:
-          <a href="${page_url}?offset=${page['previous_offset']}"
-             class="sequence-nav-button button-previous">
-            <span class="icon fa fa-chevron-left" aria-hidden="true"></span><span class="sr">${_('previous page')}</span>
-          </a>
-        %endif
-
-        ${_('Page')} ${page["page_num"]} ${_('of')} ${page["total_pages"]}
-
-        %if page["next_offset"] is not None:
-          <a href="${page_url}?offset=${page['next_offset']}"
-             class="sequence-nav-button button-next">
-            <span class="icon fa fa-chevron-right" aria-hidden="true"></span><span class="sr">${_('next page')}</span>
-          </a>
-        %endif
-    </span>
+    %if not ccx:
+      <span class="grade-book-footer">
+          %if page["previous_offset"] is not None:
+            <a href="${page_url}?offset=${page['previous_offset']}"
+               class="sequence-nav-button button-previous">
+              <span class="icon fa fa-chevron-left" aria-hidden="true"></span><span class="sr">${_('previous page')}</span>
+            </a>
+          %endif
+
+          ${_('Page')} ${page["page_num"]} ${_('of')} ${page["total_pages"]}
```

Comment:
You can't i18n like this!!! Different languages reorder nouns and adjectives. Please read completely the guide here: http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html

Please also consider watching https://www.youtube.com/watch?v=UOOMFGURr5I

@pdpinch Can you please make sure your team is properly trained in internationalization, and that you guys are prepped to catch these bugs? DB made similar comments on a separate PR, and I'm concerned this isn't making it through.